assets/css/style.css

```diff
@@ -3538,4 +3538,47 @@ button.back-to-top {
   color: #f7b205;
   font-weight: 400;
   transition-duration: 500ms;
-}
\ No newline at end of file
+}
+
+/* Pop-up poster styles */
+.poster-modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 999999; /* Sit on top */
```

Comment:
avoid doing 9999, it becomes harder to debug in future. We can skip this for now but just keep this in mind for future